src/WebAppDIRAC/WebApp/handler/MonitoringHandler.py

```diff
@@ -55,21 +55,19 @@ def web_getSelectionData(self, type):
             length = len(retVal["Value"][record])
             if length > 10000:
                 records[record] = retVal["Value"][record][length - 5000 :]
-                message = (
-                    "The %s accounting type contains to many rows: %s - > %d. Note: Only 1000 rows are returned!"
-                    % (type, record, length)
-                )
+                message = f"The {monitoringType} monitoring type contains too many rows: {record} -> {length}"
+                message += " Note: Only 1000 rows are returned!"
```

Comment:
```suggestion
                message += " Note: Only 5000 rows are returned!"
```
That's what the slicing in line 57 does, right?